apps/newsletters-api/open-api.yaml

```diff
@@ -144,3 +205,14 @@ components:
           example: '#DCDCDC'
         imageUrl:
           type: string
+    ErrorResponse:
+      type: object
+      required:
+        - ok
+      properties:
+        ok:
+          type: boolean
+          example: false
+        message:
+          type: string
+          example: No newsletter with id fake-newsletter-id
```

Comment:
For an error response, `results` is `undefined` so the field isn't included in the JSON response, but this would change if we go with using an empty array for the results field on an error.